geocoding/location-service/src/integration/mapbox_client.py

```diff
@@ -2,15 +2,24 @@
 import requests
 
 
-def mapbox_geocode(access_token, query, types=None, languages=None, limit=None, country=None):
+class EmptyContext:
```

Comment:
I think this is the same as https://docs.python.org/3.7/library/contextlib.html#contextlib.nullcontext